inc/session.class.php

```diff
@@ -1396,4 +1396,15 @@ static function getImpersonatorId() {
    static function mustChangePassword() {
       return array_key_exists('glpi_password_expired', $_SESSION);
    }
+
+   /**
+    * Get active entity id.
+    *
+    * @since 9.5
+    *
+    * @return int
+    */
+   public static function getActiveEntity() {
+      return $_SESSION['glpiactive_entity'];
+   }
```

Comment:
New method to avoid "guessing" the correct session key and benefit from auto-completion.